src/styles/reset.css

```diff
@@ -0,0 +1,4 @@
+* {
+  margin: 0;
```

Comment:
You should use `margin` for all tags you need. For example:
```
body,
h1,
p,
etc... {
  margin: 0;
}
```
But not inside `*` selector.